sorc/checkout.sh

```diff
@@ -26,7 +26,8 @@ if [[ ! -d fv3gfs.fd ]] ; then
     rm -f ${topdir}/checkout-fv3gfs.log
     git clone https://github.com/ufs-community/ufs-weather-model fv3gfs.fd >> ${topdir}/checkout-fv3gfs.log 2>&1
     cd fv3gfs.fd
-    git checkout GFS.v16.0.14
+    git checkout d021e7b0395ccac2b7a30b414b58a8c924d2784f
+####    git checkout GFS.v16.0.14
```

Comment:
Is there no tag for this? Can one be made at that hash perhaps?

Please remove the comment line on line 30.